From 6a452eef766297015ae1739409386d9e7f80df26 Mon Sep 17 00:00:00 2001
From: Jiping Ma <jiping.ma2@windriver.com>
Date: Thu, 14 Jun 2018 09:48:20 +0000
Subject: [PATCH] Buildissue: build guest qemu failed
 kernel/power/suspend.c:100:2: error: implicit declaration
 of function 'rawspin_unlock_irq'

commit [0d29972245] introduced the breakage.

Signed-off-by: Jiping Ma <jiping.ma2@windriver.com>
---
 kernel/power/suspend.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/power/suspend.c b/kernel/power/suspend.c
index 3dc9355..102c120 100644
--- a/kernel/power/suspend.c
+++ b/kernel/power/suspend.c
@@ -97,7 +97,7 @@ static void freeze_enter(void)
 
  out:
 	suspend_freeze_state = FREEZE_STATE_NONE;
-	rawspin_unlock_irq(&suspend_freeze_lock);
+	raw_spin_unlock_irq(&suspend_freeze_lock);
 }
 
 void freeze_wake(void)
-- 
1.7.9.5

